### PR TITLE
MC-1775 - migration guide about the changed property

### PR DESCRIPTION
**Pull request by promanenko**

### CODE REVIEW

File: docs/modules/ROOT/pages/migration-guides.adoc [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+
 == Changes in 5.1.x
 
- * The `mc-conf.sh` command was renamed to `hz-mc conf`
- * The `start.sh` was deprecated and is superseded by the `hz-mc start` command
- * The supported way for starting up a standalone Management Center instance was changed from `java -jar ...` command to `hz-mc start`
- * The Console within Management Center is disabled by default and has to be enabled within the member's configuration. See xref:tools:console.adoc[`Executing Console Commands`]
+ * The `mc-conf.sh` command has been renamed to `hz-mc conf`.
+ * The `start.sh` has been deprecated and is superseded by the `hz-mc start` command.
+ * The supported way for starting up a standalone Management Center instance has been changed from `java -jar ...` command to `hz-mc start`/
```
Comment (zhmaeff) [inline docs/modules/ROOT/pages/migration-guides.adoc:12]:
```suggestion
 * The supported way for starting up a standalone Management Center instance has been changed from `java -jar ...` command to `hz-mc start`
```

File: docs/modules/ROOT/pages/migration-guides.adoc [hunk 2]
```diff
@@ -1,12 +1,16 @@
-= Upgrading to Management Center 5.1.x
+= Upgrading to Management Center 5.2.x
 [[migration-guides]]
 
+== Changes in 5.2.x
+
+* The custom Log4j configuration file property `log4j.configurationFile` has been changed to `logging.config`.
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/migration-guides.adoc:6]: Assuming the property functionality being the same, maybe it is better to say "has been renamed as"

**Review by zhmaeff — APPROVED** [review]

File: docs/modules/ROOT/pages/migration-guides.adoc (hunk 2 above)
Comment (promanenko) [reply docs/modules/ROOT/pages/migration-guides.adoc:6]: it's not actually our choice, but the change caused by the framework we use.
I think "renamed" as sounds more like we deliberately did it. WDYT?

Comment (Serdaro) [reply docs/modules/ROOT/pages/migration-guides.adoc:6]: Ah okay.. we can leave it as it is then. Thank you Pavlo

**Pull request merged**